hack/util.sh

```diff
@@ -460,17 +465,17 @@ function kill_all_processes()
 {
 	local sudo="${USE_SUDO:+sudo}"
 
-	pids=($(jobs -pr))
-	for i in ${pids[@]-}; do
-		pgrep -P "${i}" | xargs $sudo kill &> /dev/null
-		$sudo kill ${i} &> /dev/null
+	pids="$(jobs -pr)"
+	for i in ${pids}; do
+		pgrep -P "${i}" | xargs "${sudo}" kill &> /dev/null
```

Comment:
Here likewise we don't really need the `xargs` if we just loop over PIDs in the first place